components/Products/ProductUpload.js

```diff
@@ -0,0 +1,177 @@
+import { useRef, useState } from "react";
+import supabase from "../../utils/supabaseClient";
+import { useRouter } from "next/router";
+
+function ProductUpload() {
+  const [imageSrc, setImageSrc] = useState("");
+
+  const productImg = useRef();
+  const category = useRef();
+  const productName = useRef();
+  const description = useRef();
+  const condition = useRef();
+  const router = useRouter();
+
+  async function handleProductSubmit(event) {
+    event.preventDefault();
+
+    const selectedProduct = productImg.current.files[0];
+    const enteredCategory = category.current.value;
+    const enteredProductName = productName.current.value.toLowerCase();
+    const enteredDescription = description.current.value;
+    const enteredCondition = condition.current.value;
+    // const altText = `Image of Product ${selectedProduct.name}`;
+
+    const formData = new FormData();
+    formData.append("file", selectedProduct);
+    formData.append("upload_preset", "vwz7spwe");
+
+    // upload picture to cloudinary
+    const API_ENDPOINT =
+      "https://api.cloudinary.com/v1_1/streetcred/image/upload";
+    const options = {
+      method: "POST",
+      body: formData,
+    };
+    const cloudinary = await fetch(API_ENDPOINT, options).then((response) => {
+      console.log("picture is uploaded to cloudinary");
```

Comment:
we might want to remove the `console.log` ?

Reply:
it's done